Sources/SpeziAccountPhoneNumbers/DataEntry/OTCEntryView.swift

```diff
@@ -120,36 +80,62 @@ struct OTCEntryView: View {
         }
     }
     
+    private var formattedTextField: some View {
+        TextField("", text: $verificationCode)
+            .focused($isTextFieldFocused)
+            .opacity(0)
+            .multilineTextAlignment(.center)
+            .frame(width: CGFloat(codeLength) * 60, height: 45)
+            .background(
+                HStack(spacing: 15) {
+                    ForEach(0..<codeLength, id: \.self) { index in
+                        pinBackground(index: index)
+                    }
+                }
+            )
+            .overlay(
+                HStack(spacing: 15) {
+                    ForEach(0..<codeLength, id: \.self) { index in
+                        pinDigit(index: index)
+                    }
+                }
+            )
+            .accessibilityLabel("Verification code entry")
+#if !os(macOS)
+            .keyboardType(.numberPad)
+            .textContentType(.oneTimeCode)
+#endif
+            .onChange(of: verificationCode) { _, newValue in
```

Comment:
We probably also need to handle the case what happens if a user enters more digits that the ones provided and/or if we paste in a code that is longer than the expected one time code?

Reply:
I think the current implementation already handles this with `verificationCode = String(filtered.prefix(codeLength))`.

Reply:
Nice, missed that; thank you!